videoroom/2_EnvironmentPreparation.md

```diff
@@ -95,10 +97,12 @@ mix deps.get
 npm ci --prefix=assets
 ```
 
+You also need to find your local IP address. The way to do that is described [here](https://github.com/membraneframework/membrane_videoroom#launching-of-the-application-1).
```

Comment:
How about adding an information that we don't need to set this in docker environment? There is no need for the user to follow the linked instruction while running the application in the docker container.